mcp-test/test-assistant-dummy.mjs

```
@@ -0,0 +1,299 @@
+#!/usr/bin/env node
+
+// Test script with dummy SDK functions to figure out MCP integration
+import { experimental_createMCPClient, generateText, streamText } from 'ai';
+import { openai } from '@ai-sdk/openai';
+import dotenv from 'dotenv';
+import { resolve } from 'path';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)